qa/L0_config/config_generator.py

```diff
@@ -20,6 +20,7 @@ def _get_sweep_configs(profile_models):
 
     sweep_configs = []
     model_config = {
+        'metrics': ['cpu_available_ram', 'cpu_used_ram'],
```

Comment:
So in the actual config, this list of metrics works out to be just the "extra metrics" that are being collected. I don't know if there is a good case to have this be called `metrics` anymore.

Reply:
I could use the full list, but since the minimum list is added on to the provided list, I just keep it short.
Maybe we can call it `optional_metrics`?